.github/ISSUE_TEMPLATE.md

```diff
@@ -13,31 +13,14 @@ What did you expect to see when you do...?
 What actually happend? Why was it incorrect?
 
 
+<!-- if this is a feature request, you can ignore this next part -->
+
 ### Steps to reproduce behavior
 
 Please fill out as much information as possible.
 
 
-### Environment?
-
-Please add infomation about your environment.
-
-#### OS distribution and version:
-
-```
-$ cat /etc/os-release 
```

Comment:
I remove this, because its not always necessary/needed, and when it is, we can just ask for it. Otherwise it seems like its too much to fill out. Same for `go version`...

Reply:
I think a large proportion of issues are OS or at least kernel version dependent. I'm not so sure about removing it.

Reply:
Okay, I'll add this part back.